app/types/index.ts

```diff
@@ -0,0 +1,23 @@
+import {RequestObj} from './../request_list'
+export type POPUP_PROPS = {
```

Comment:
Why a type instead of an interface here?

Reply:
Right. Fixed that. Thanks!